test/test_helper.rb

```diff
@@ -1,12 +1,16 @@
 ENV['RAILS_ENV'] ||= 'test'
 require File.expand_path('../../config/environment', __FILE__)
 require 'rails/test_help'
+require 'minitest/stub_any_instance'
 
 class ActiveSupport::TestCase
 
   VCR.configure do |config|
     config.cassette_library_dir = "fixtures/cassettes"
     config.hook_into :webmock
+    config.ignore_request do |request|
+      URI(request.uri).host == '127.0.0.1'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.